src/protagonist/Engine.Tests/Integration/ImageIngestTests.cs

```diff
@@ -269,6 +323,59 @@ public async Task IngestAsset_Error_ExceedAllowance()
         storage.Should().BeNull();
     }
     
+    [Fact]
+    public async Task IngestAsset_Error_ExceedAllowanceOnReingest()

```

Comment:
What causes this to fail when the other new test passes?

Reply:
it's to do with the size of the previous asset in the db - on success it's 950, with an overall storage of 950, a max of 1000 and a 'new' asset size of 1000, meaning the logic is 950 - 950 + 1000 <= 1000 

in the failure test it's 500 with an overall storage of 950, so it produces 950 - 500 + 1000 <= 1000